style.css

```diff
@@ -0,0 +1,524 @@
+  border-radius: 3px;
+  font-size: 12px;
+  font-weight: 650;
+}
+
+.comment-and-icons {
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  width: 600px;
+  height: 80px;
+  padding-top: 10px;
+  padding-left: 20px;
+  padding-right: 30px;
+}
+
+.comment-icons i {
+  padding-left: 20px;
+}
+
+/* newsletters section */
+
+/* newslatter section */
+
+.newsletters {
+  display: flex;
+  flex-wrap: wrap;
+  width: 1000px;
+  height: 880px;
+  margin: 45px 193px 0 193px;
+  padding: 20px 50px 0 50px;
+  font-size: 16px;
+  line-height: 16px;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.newsletters img {
+  width: 220px;
+  height: 131px;
+  padding-left: 15px;
+  padding-top: 15px;
+}
+
+.news-more {
+  width: 820px;
+  height: 500px;
+  display: grid;
+  grid-template-columns: 25% 25% 25%;
+  column-gap: 50px;
+}
+
+.news1 {
+  width: 272px;
+  height: 250px;
+  padding-left: 35px;
+}
+
+.news-p {
+  padding-left: 25px;
+}
+
+.news-more2 {
+  display: flex;
+  width: 272px;
+  padding-top: 30px;
+}
+
+.news2 {
+  width: 272px;
+  height: 250px;
+  padding-left: 35px;
+}
+
+.most-like {
+  display: flex;
+  position: absolute;
+  padding: 20px 20px;
+  top: 6600px;
+  right: 70px;
+  width: 200px;
+  height: 650px;
+}
+
+.news-more3-border {
+  border-top: 1px solid silver;
+  padding: 10px 10px;
+  margin: 5px;
+}
+
+.news-more3-border a {
+  text-decoration: none;
+  color: black;
+}
+
+/* footer section */
+.footer-design {
+  font-size: 16px;
+  line-height: 16px;
+  color: black;
+  display: flex;
+  width: 90%;
+  height: 430px;
+  margin: auto;
+  justify-content: space-between;
+  padding: 10px 30px 10px 25px;
+  border-top: 1px solid silver;
+}
+
+.left-footer {
+  width: 200px;
+  height: 25px;
+  float: left;
+}
+
+#logo-footer {
+  width: 200px;
+}
+
+.right-footer {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7520px;
+  left: 1150px;
+}
+
+.footer3 {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7550px;
+  left: 100px;
+  line-height: 23px;
+}
+
+.footer4 {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7550px;
+  left: 300px;
+  line-height: 23px;
+}
+
+.footer5 {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7550px;
+  left: 500px;
+  line-height: 23px;
+}
+
+.footer6 {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7550px;
+  left: 700px;
+  line-height: 23px;
+}
+
+.footer7 {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7550px;
+  left: 900px;
+  line-height: 23px;
+}
+
+.footer8 {
+  display: flex;
+  position: absolute;
+  width: 250px;
+  top: 7550px;
+  left: 1100px;
+  line-height: 23px;
+  border-left: 1px solid silver;
+}
+
+.footer3 ul {
+  list-style: none;
+}
+
+.footer3 a {
+  color: black;
+  text-decoration: none;
+}
+
+.footer4 ul {
+  list-style: none;
+}
+
+.footer4 a {
+  color: black;
+  text-decoration: none;
+}
+
+.footer5 ul {
+  list-style: none;
+}
```

Comment:
Same thing happening here on the footerx ul (i.e. x = 3,4,5,6,7,8).

All have the same list-style property.